src/modules/wasi/x86-64-linux/WspOneModule-x86-64-linux.v3

```diff
@@ -147,6 +163,136 @@ class X86_64Linux_WspOneModule extends WspOneModule {
 		DataWriters.write_range_i64(range.result, t.0);
 		return HostResult.Value1(Values.I32_0);
 	}
+        def path_unlink_file(args: Range<Value>) -> HostResult {
+		// 0=fd 1=pathptr 2=length
+		var fd		= Values.unbox_i(args[0]);
+		var pathptr	= args[1];
+		var pathlen	= args[2];
+
+		var dir_fd = fdmap.get(fd);
+		if (dir_fd == null) return HostResult.Value1(Values.box_i(WspOneErrno.BADF.code));
```

Comment:
This line (which is a copied a few times from a previous example) can use the `i` helper, or even a constant factored out to the top-level.